Windows/ppsspp.rc

```diff
@@ -445,6 +445,13 @@ BEGIN
         MENUITEM "Memory View...",                          ID_DEBUG_MEMORYVIEW
     END
 
+    POPUP "Movie"
+    BEGIN
+        MENUITEM "Dump Frames",                             ID_MOVIE_DUMPFRAMES

```

Comment:
Hmm, maybe "Record Display" or "Export Screen" would be more clear?  Sorry to nitpick naming.  "Dump frames" makes me think it's going to individually export each frame to a separate file.

-[Unknown]